engine/internal/processor/processor.go

```diff
@@ -253,12 +263,13 @@ func (p *P) processTask(
 		return fmt.Errorf("build request: %s", err)
 	}
 
-	log.Printf("Sending request to the LLM server: %s\n", req.URL)
+	log.Info(fmt.Sprintf("Sending request to the LLM server: %s", req.URL))
```

Comment:
Since it's a structured logging library, it's common practice to split the message into key-value pairs for variables.
```suggestion
	log.Info("Sending request to the LLM server", "url", req.URL)
```

Reply:
got it!